buildenv/jenkins/README.md

```diff
@@ -27,42 +27,67 @@ This folder contains Jenkins pipeline scripts that are used in the OpenJ9 Jenkin
+- Current supported test groups are functional & system
+- Current available platforms are
+    - Linux on x86-64
+        - Spec: x86-64_linux
+        - Shortname: xlinux
+    - Linux on x86-64 largeheap/non-compressed references
+        - Spec: x86-64_linux_xl
+        - Shortname: xlinuxlargeheap or xlinuxxl
+    - Linux on x86-64 with CMake
+        - Spec: x86-64_linux_cm
+        - Shortname: xlinuxcm or xlinuxcmake
+    - Linux on s390x
+        - Spec: s390x_linux
+        - Shortname: zlinux
+    - Linux on ppc64le
+        - Spec: ppc64le_linux
+        - Shortname: plinux
+    - AIX on ppc64
+        - Spec: ppc64_aix
+        - Shortname: aix
+    - Windows on x86-64
+        - Spec: x86-64_windows
+        - shortname: win
+    - Windows on x86 (32bit, supported on JDK8 only)
+        Spec: x86-32_windows
+        Shortname: win32
+    - OSX on x86-64
+        - Spec: x86-64_mac
+        - Shortname: osx
+    - OSX on x86-64 largeheap/non-compressed references
+        - Spec: x86-64_mac_xl
+        - Shortname: osxlargeheap or osxxl
+
 - OpenJ9 committers can request builds by commenting in a pull request
-    - Format: `Jenkins <build type> <level> <platform>[,<platform>,...,<platform>] jdk<version>[,jdk<version>,...,jdk<version>]`
+    - Format: `Jenkins <build type> <level>.<group> <platform>[,<platform>,...,<platform>] jdk<version>[,jdk<version>,...,jdk<version>]`
     - `<build type>` is compile | test
     - `<level>` is sanity | extended (required only for "test" `<build type>`)
-    - `<platform>` is aix | xlinux | xlinuxlargeheap | zlinux | plinux | win | win32
-    - `<version>` is the number of the supported release, e.g. 8 | 11 | n 
-- Note: You can use keyword `all` for platform
+    - `<group>` is functional | system
+    - `<platform>` is one of the platform shorthands above
+    - `<version>` is the number of the supported release, e.g. 8 | 11
+- Note: You can use keyword `all` for platform but not for test level/type or JDK versions.
+- Note: For backward compatability `<level>.<test type>` equal to `sanity` or `extended` is acceptable and will map to `sanity.functional` and `extended.functional` respectively.
 
 ###### Examples
 - Request a Compile-only build on all platforms and multiple versions by commenting in a PR
     - `Jenkins compile all jdk8,jdk11`
-- Request a Sanity build on zLinux and multiple versions
-    - `Jenkins test sanity zlinux jdk8,jdk11`
-- Request an Extended build on pLinux for a single version
-    - `Jenkins test extended plinux jdk8`
-- Request a Sanity build on z,p Linux for multiple versions
+- Request a sanity functional build on zLinux and multiple versions
+    - `Jenkins test sanity.functional zlinux jdk8,jdk11`
+- Request an extended functional and system build on pLinux for a single version
+    - `Jenkins test extended.functional,extended.system plinux jdk8`
+- Request a sanity build on z,p Linux for multiple versions
     - `Jenkins test sanity zlinux,plinux jdk8,jdk9`
```

Comment:
How about removing jdk9 here since it is obsolete.

Reply:
Done